client/src/components/TextInputComponent/TextInput.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./TextInput.css";
+
+const TextInput = props => (
+  <>
+    <label className="label-name">{props.label}</label>
+    <input
+      className="text-input"
+      type={props.type}
+      placeholder={props.placeholder}
+    />
+  </>
+);
```

Comment:
Can you add `value` and `onChange`? We want all inputs to be controlled. You can read more [here](https://reactjs.org/docs/forms.html)

Reply:
You'll need to define a simple `onChangeHandler` function in the `TextInput.stories.js` file and pass it down as a prop

Reply:
I read about it and implemented value and onChange.  I feel REALLY dumb that I didn't do that before,  React is definitely my weakest link in skill and knowledge. So i'm glad i'm getting to build this out completely.  Thank you for the hand holding, its allowing me to understand the steps clearly and won't need to repeat asking the same topics over and over.